baselines/fedmix/fedmix/conf/cifar10.yaml

```diff
@@ -0,0 +1,33 @@
+---
+
+seed: 42
+device: cuda
```

Comment:
This should be used only for the server (i.e. using this `device` to evaluate the global model). On the other hand, clients should pick the device the Flower Simulation Engine exposes to them (which is controlled with the `client_resources` dict passed to `start_simulation`. See my related comments in the `main.py` and `client.py`

Reply:
With [this](https://github.com/adap/flower/pull/2213#discussion_r1337194346) comment resolved, this is fixed as well, right?

Reply:
Yes, you are totally right 💯 .  How about renaming this to `server_device`?

Reply:
Sure, that makes more sense.